### PR TITLE
add s1meta.geoloc

### PR DESCRIPTION
**Pull request by agrouaze**
add s1meta.geoloc (second try because first PR #45 ) was linked to a branch heading on a branch with lots of changes and not the latest develop.
PR meant to address issue #42 .

### CODE REVIEW

**Comment by agrouaze** [comment]
image

**Comment by agrouaze** [comment]
polygon of the footprint given by `s1meta.geoloc.footprint`
image

File: src/xsar/sentinel1_xml_mappings.py [hunk 1]
```diff
@@ -121,10 +122,19 @@ def list_poly_from_list_string_coords(str_coords_list):
     'annotation': {
         'atrack': (uniq_sorted, '/product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/line'),
         'xtrack': (uniq_sorted, '/product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/pixel'),
+        'atrack_grid': (int_array, '//product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/line'),
+        'xtrack_grid': (int_array, '//product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/pixel'),
```
Comment (oarcher) [inline src/xsar/sentinel1_xml_mappings.py:126]: I think that an absolute xpath should be fine ('/product', not '//product')

**Review by oarcher — CHANGES_REQUESTED** [review]

File: src/xsar/sentinel1_xml_mappings.py (hunk 1 above)
Comment (agrouaze) [reply src/xsar/sentinel1_xml_mappings.py]: correction done!

**Comment by agrouaze** [comment]
with last commit we temporarily remove `incidence` and `elevation` from `s1meta.geoloc`.
image

**Review by oarcher — APPROVED** [review]

**Pull request merged**